models/intel/action-recognition-0001/description/action-recognition-0001.md

```diff
@@ -52,18 +49,17 @@ The action-recognition-0001-decoder model accepts stack of frame embeddings, com
 | GFlops                          | 0.147                                     |
 | MParams                         | 4.405                                     |
 
-
 ### Inputs
 
-1. name: "0" , shape: [1x16x512] - An embedding image in the format [BxTxC],
-   where:
-    - B - batch size.
-    - T - Duration of input clip.
-    - C - dimension of embedding.
+1. name: `0` , shape: `1, 16, 512` - An embedding image in the format `B, T, C` where:
+
+- `B` - batch size
+- `T` - duration of input clip
+- `C` - dimension of embedding
 
 ### Outputs
 
-The model outputs a tensor with the shape [bx400], each row is a logits vector of performed actions.
+The model outputs a tensor with the shape `b, 400`, each row is a logits vector of performed actions.
```

Comment:
do we really need this dynamism with `b` or batch size is matched with input batch size?

Reply:
Updated